nixos.nix

```
@@ -506,9 +506,9 @@ in
       in
       foldl' recursiveUpdate { } (map mkPersistFileService files);
 
-    fileSystems = bindMounts;
+    fileSystems = mkIf (directories != [ ]) bindMounts;
```

Can also be written as
```suggestion
    fileSystems = mkIf (bindMounts != { }) bindMounts;
```